awswrangler/s3/_write_parquet.py

```diff
@@ -39,12 +39,18 @@ def _get_file_path(file_counter: int, file_path: str) -> str:
 def _new_writer(
     file_path: str,
     compression: Optional[str],
+    pyarrow_additional_kwargs: Optional[Dict[str, str]],
     schema: pa.Schema,
     boto3_session: boto3.Session,
     s3_additional_kwargs: Optional[Dict[str, str]],
     use_threads: bool,
 ) -> Iterator[pyarrow.parquet.ParquetWriter]:
     writer: Optional[pyarrow.parquet.ParquetWriter] = None
+    if not pyarrow_additional_kwargs:
+        pyarrow_additional_kwargs = {}
```

Comment:
Would `pyarrow_additional_kwargs: Optional[Dict[str, str]] = {},` as a default work on line 42? This way we don't need this logic here